[new-product] meilisearch (#4372)
captn3m0/endoflife.date: products/meilisearch.md

---
title: Meilisearch
category: server-app
tags: meilisearch
iconSlug: meilisearch
permalink: /meilisearch
versionCommand: meilisearch --version
releasePolicyLink: https://www.meilisearch.com/docs/learn/update_and_migration/versioning

> changelogTemplate: https://github.com/meilisearch/meilisearch/releases

Reviewer: ```suggestion
changelogTemplate: https://github.com/meilisearch/meilisearch/releases/tag/v__LATEST__
```

This generate a direct leak to latest release for each release cycle.

releaseDateColumn: true

identifiers:
-   repology: meilisearch

auto:
-   git: https://github.com/meilisearch/meilisearch.git

> releases:

Reviewer: A bit more release cycle would be great. Is it possible to document all minor since 1.0 ?

Author: I added older releases

> releases:

Reviewer: ```suggestion
# eol(x) = releaseDate(x+1)
releases:
```

For future maintainer (based on https://www.meilisearch.com/docs/learn/update_and_migration/versioning#support-for-previous-versions)

-   releaseCycle: "1.5"
    releaseDate: 2023-11-20
    eol: false
    latest: "1.5.1"
    latestReleaseDate: 2023-12-13

---

> > [Meilisearch](https://www.meilisearch.com/) is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users.

Reviewer: It's a bit too much commercial IMO. Could you rewrite this so this is more factual / neutral. Something like :

> [Meilisearch](https://www.meilisearch.com/) is a fast and scalable search engine that enables developers to integrate search capabilities into their websites and applications.

Author: Changed the description

> Meilisearch releases and their associated binaries are available [in this GitHub page](https://github.com/meilisearch/meilisearch/releases).

Reviewer: Such things are rarely documented on endoflife.date.

I don't mind keeping it if you think it's an important information, but it should be moved to the end IMO (the important part is to explain the release / support policy).

Author: Removed

> The binaries are versioned following [SemVer conventions](https://semver.org/). To know more, read our [versioning policy](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).

Reviewer: ```suggestion
Meilisearch [follows semver](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
Minor versions release cadence is between four and six times a year.
Only the the latest engine release is maintained with high-priority bug and security fixes.
```

This description is missing the release cadence and support policy.
The suggestion is based on what I read on https://www.meilisearch.com/docs/learn/update_and_migration/versioning.

> 

Reviewer: From what I read on https://www.meilisearch.com/docs/learn/update_and_migration/versioning, it would be good to also:

- add a sentence to says that [Meilisearch engine version numbers have no relationship to SDK version numbers](https://www.meilisearch.com/docs/learn/update_and_migration/versioning#sdk-versioning),
- and maybe talk about the cloud offer ?

Author: I have added two sentences about the SDK and the cloud

> Differently from the binaries, crates in this repository are not currently available on [crates.io](https://crates.io/) and do not follow [SemVer conventions](https://semver.org).

Reviewer: Considering Meilisearch is distributed as a binary package, I don't understand this sentence.

Author: Crates are the rust packages, or what exactly do you mean?

Reviewer: I don't understand this sentence, and why crates are mentioned. Do you mean the Meilisearch engine can also be installed that way (through crates) ?

Author: Changed the description

---
title: Meilisearch
category: server-app
tags: meilisearch
iconSlug: meilisearch
permalink: /meilisearch
versionCommand: meilisearch --version
releasePolicyLink: https://www.meilisearch.com/docs/learn/update_and_migration/versioning
changelogTemplate: https://github.com/meilisearch/meilisearch/releases/tag/v__LATEST__
releaseDateColumn: true

identifiers:
-   repology: meilisearch

auto:
-   git: https://github.com/meilisearch/meilisearch.git

# eol(x) = releaseDate(x+1)
releases:
-   releaseCycle: "1.5"
    releaseDate: 2023-11-20
    eol: false
    latest: "1.5.1"
    latestReleaseDate: 2023-12-13

-   releaseCycle: "1.4"
    releaseDate: 2023-09-25

>     eol: true

Reviewer: Must be set to 1.5 `releaseDate`. Same for all other releases.
